src/Control/Distributed/Process/Node/RegistryAgent.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
A single module for a single function sounds overkill. I'd inline `registryMonitorAgent` into `Node.hs`.